swift/Tests/Blockchains/DashTests.swift

```diff
@@ -0,0 +1,38 @@
+// Copyright © 2017-2018 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import TrustWalletCore
+import XCTest
+
+class DashAddressTests: XCTestCase {
+    func testAddress() {
+        let privateKey = PrivateKey(wif: "XDoxFwfxsEZDd15uNyj8vt64c3GLxcFjTefnUz7gckvAJeYSFaRz")!
+        let publicKey = privateKey.getPublicKey(compressed: true)
+        let address = Dash().address(for: publicKey)
+
+        XCTAssertEqual(address.description, "Xw7HTXGY3TFeA3ZsVuMRrYh96GtwWb4hQb")
+    }
+
+    func testExtendedKeys() {
+        let wallet = HDWallet(mnemonic: "ripple scissors kick mammal hire column oak again sun offer wealth tomorrow wagon turn fatal", passphrase: "TREZOR")
```

Comment:
`ripple scissors kick mammal hire column oak again sun offer wealth tomorrow wagon turn fatal` is used in 4 places, can we create one class for it?

Reply:
we can create a factory method for that

Reply:
👍